### PR TITLE
Keep styles

### PR DESCRIPTION
**Pull request by jasongotlieb**
this includes the fix found in other PR as well. that one should be merged first.

### CODE REVIEW

File: src/multiple_value/multiple_value.js [hunk 1]
```diff
@@ -166,8 +168,8 @@ class MultipleValue extends React.PureComponent {
                     color={config[`style_${dataPoint.name}`]}
                     onClick={() => { this.handleClick(dataPoint, event) }}
                     layout={this.getLayout()}
-                  >
-                    {dataPoint.formattedValue}
+                  >                  
+                  { dataPoint.html ? ReactHtmlParser(dataPoint.html) : dataPoint.formattedValue }
```
Comment (bryans99) [inline src/multiple_value/multiple_value.js:172]: Hm, a little scary!

See security section - https://github.com/peternewnham/react-html-parser#readme

"All HTML should be properly sanitized using a dedicated sanitisation library"

**Review by bryans99 — CHANGES_REQUESTED** [review]

File: src/multiple_value/multiple_value.js (hunk 1 above)
Comment (jasongotlieb) [reply src/multiple_value/multiple_value.js]: Ahh. Good catch. Thanks. Added an sanitizer.

**Comment by AnnaSerova** [comment]
@bryans99 Does it look good now?

File: src/multiple_value/multiple_value_container.js [hunk 2]
```diff
@@ -57,7 +60,9 @@ looker.plugins.visualizations.add({
 
     if(data.length < 1) {
       this.addError({title: "No Results"})
+      this.chart = renderBlankVisualization(element)
```
[the remark this answers is not in the document]
Comment (jasongotlieb) [reply src/multiple_value/multiple_value_container.js]: Hey. I believe I've addressed this comment now, too! Let me know what you think.

**Comment by bryans99** [comment]
Thanks all

**Pull request merged**